client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,67 @@
+.scoreboard {
+    border: 2px solid black;
+    box-sizing: border-box;
+    background: #fff;
+    border-width: 2px;
+    border-radius: 8px;
+    display: flex;
```

Comment:
This isn't necessary and I think is shrinking the table. Removing this I can scroll to the very top of the scoreboard on https://test.virtualtabletop.io/PR-1381/Roger

Reply:
Yep. Thanks.